tests/tasks/setup_candlepin.yml

```
@@ -0,0 +1,89 @@
+---
+- name: Get LSR_RHC_TEST_DATA environment variable
+  set_fact:
+    lsr_rhc_test_data_file: "{{ lookup('env', 'LSR_RHC_TEST_DATA') }}"
+  delegate_to: 127.0.0.1
+  become: false
+
+- name: Import test data
+  include_vars:
+    file: "{{ lsr_rhc_test_data_file }}"
+  when: lsr_rhc_test_data_file | length > 0
+  delegate_to: 127.0.0.1
+  become: false
+
+- name: Deploy Candlepin
+  block:
+    - name: Clone ansible-role-candlepin repo
+      git:
+        repo: https://github.com/candlepin/ansible-role-candlepin.git
+        version: master
+        dest: /tmp/candlepin-role-repo
+      delegate_to: 127.0.0.1
+      become: false
+
+    - name: Create role symlink
+      file:
+        src: "/tmp/candlepin-role-repo/"
+        dest: "{{ playbook_dir }}/roles/candlepin"
+        state: link
+      delegate_to: 127.0.0.1
+      become: false
+
+    - name: Create empty/fake rvm.ruby module
+      file:
+        dest: "{{ playbook_dir }}/roles/rvm.ruby"
+        state: directory
```

```suggestion
        state: directory
        owner: root
        group: root
        mode: "0777"
```
to satisfy ansible-lint error

---

hm is root:root correct here? this will be created in my own source directory